### PR TITLE
[xbmc][WindowsStore] Set store minimum version

### PR DESCRIPTION
**Pull request by Paxxi**
This Specifies the minimum runtime version of Win10 that we support as koying figured out.
For now this won't do anything at all but once we bump the sdk to the latest version it's important to keep this at least one version behind unless we really need something from the latest version
## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by afedchin — APPROVED** [review]
If we want to support windows mobile on most devices, we will stay on 14393 as min for long time. because only 13 models received creators update.

**Comment by Rechi** [comment]
Is the following unrelated to the version you set in this PR or should they stay the same?
https://github.com/xbmc/xbmc/blob/ce76a6d26cfbe6491f5542441a130ebb8aff1809/tools/windows/packaging/uwp/package.appxmanifest.in#L19

Also keep in mind this PR currently contains an unrelated commit (vswhere).

**Comment by MartijnKaijser** [comment]
If we need to start defining the version at several places it make make sense to put it in version.txt or something

**Comment by Paxxi** [comment]
Didn't know about that one. According to https://docs.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-targetdevicefamily it looks like they should match.
I'll see about having it in a version file so we have it in one place. 

Might want to do the same for the max version so that it matches the sdk used in buildsetup.bat. Not sure if that would cause any issues whenever a new windows is released though.

**Comment by afedchin** [comment]
@Paxxi I think we should define min SDK ver in ArchSetup.cmake, at least for do not changing CMakeList.txt for every version bump.

@MartijnKaijser  no need to define this in version.txt because it's not a target SDK version but minimum supported, this may no change for couple years.

File: tools/windows/packaging/uwp/package.appxmanifest.in [hunk 1]
```diff
@@ -16,7 +16,7 @@
   </Properties>
 
   <Dependencies>
-    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.0.0" MaxVersionTested="10.0.0.0" />
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="@VS_MINIMUM_SDK_VERSION@" MaxVersionTested="10.0.0.0" />
```
Comment (MartijnKaijser) [inline tools/windows/packaging/uwp/package.appxmanifest.in]: Guess we should also bump MaxVersionTested to at least MinVersion? :)

File: tools/windows/packaging/uwp/package.appxmanifest.in [hunk 2]
```diff
@@ -16,7 +16,7 @@
   </Properties>
 
   <Dependencies>
-    <TargetDeviceFamily Name="Windows.Universal" MinVersion="@VS_MINIMUM_SDK_VERSION@" MaxVersionTested="10.0.0.0" />
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="@VS_MINIMUM_SDK_VERSION@" MaxVersionTested="@VS_MAXIMUM_SDK_VERSION@" />
```
Comment (afedchin) [inline tools/windows/packaging/uwp/package.appxmanifest.in]: please remove this change. VS will change this as needed

Comment (afedchin) [reply tools/windows/packaging/uwp/package.appxmanifest.in]: FYI you can download appx from the mirrot and check `AppxManifest.xml` inside

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by MartijnKaijser** [comment]
jenkins build this please

**Comment by Rechi** [comment]
has to be squashed

**Review by Rechi — COMMENTED** [review]
I can't say anything about this.

**Pull request merged**